README.md

```diff
@@ -1,5 +1,168 @@
-# pd-achievements
+# 🏆 Playdate Achievements
 
-A cross-game achievement standard for the Playdate console.
+_An open achievement standard for the Playdate console._
 
-Content to be placed here soon.
+Playdate Achievements is a community project establishing an open standard for achievements in [Playdate](https://play.date)™ games. Playdate is fun! Games on Playdate are fun! Achievements in games on Playdate are fun! Having a standard ensures a consistent experience for players, and makes it possible to view achievements earned across games in one place.
+
+1. Play Playdate games!
+2. Earn achievements!
+3. View them in your Trophy Case!
+
+## Everything On Display
+
+[Trophy Case](https://github.com/gurtt/trophy-case) is a dedicated viewer which displays the achievements you’ve earned across _all_ Playdate games in one place. Trophy Case was designed in parallel with the Playdate Achievements standard, providing first-class support for its features and showcasing any games which choose to integrate with it.
+
+<div align="center">
+	<img src="./images/viewers/trophy-case-pd.png" width="400"/>
+</div>
+
+Of course, games may also present their earned achievements themselves. Additionally, the open standard makes it possible for others to create dedicated achievement viewer apps in the future.
+
+## Show Me the Games!
+
+It’s all about the games. Here’s a (non-comprehensive) list of games that award Playdate Achievements. Made your own? [Add it to the list!](./CONTRIBUTING.md)
```

Comment:
This could be a link to https://github.com/PlaydateSquad/pd-achievements#contributing for now, until there is a dedicated CONTRIBUTING page, just to unblock this. (Same below.)

Reply:
Sounds good.